### PR TITLE
Fix specs

### PR DESCRIPTION
**Pull request by mdarby**
I nixed the integration tests for :v1 as it's so deprecated at this point even I as an employee don't have access to it :)

### CODE REVIEW

**Comment by jjasghar** [comment]
:+1:

**Comment by mdarby** [comment]
Works locally 'o course. Hrm.

**Comment by martinb3** [comment]
Looks to be passing now. Is it ready for merge?

**Comment by jjasghar** [comment]
looks good to me, lets see travis do [our|my] work!

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +45,6 @@ deploy:
   on:
     tags: true
 notifications:
-    irc:
-      - "chat.freenode.net#openstack-chef"
-      - "chat.freenode.net#chef-hacking"
+  irc:
```
Comment (jjasghar) [inline .travis.yml]: Actually please remove these.

File: Rakefile [hunk 2]
```diff
@@ -14,7 +14,7 @@
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
-# limitations under the License.
+# limitations under the License.
```
Comment (jjasghar) [inline Rakefile]: WS

**Comment by jjasghar** [comment]
Couple comments.

**Comment by mdarby** [comment]
Getting such random failures. I'm afraid this test suite needs a fair amount of love.

**Comment by mdarby** [comment]
Ok, this run should fix it.

**Comment by martinb3** [comment]
@mdarby I'm seeing it show green now. Am I looking in the wrong spot?

**Comment by mdarby** [comment]
@martinb3 for some reason the Github PR status doesn't always 1:1 mirror Travis. It's fine to merge now though!

**Comment by martinb3** [comment]
Should we open an issue to track the removed tests? Just so we don't forget to put them back?

**Comment by mdarby** [comment]
issue opened

**Pull request merged**